lib/net.c

```diff
@@ -1112,6 +1123,10 @@ start_iface(struct gatekeeper_if *iface)
 		goto stop_partial;
 
 out:
+	ret = rte_eth_dev_set_mtu(iface->id, iface->mtu);
+	if (ret < 0)
```

Comment:
I think it's better to have a log message here to tell the possible reasons of the failure, as there are several possible causes.